<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout"/>
    </item>
    <item>
     <widget class="QFrame" name="searchFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Find</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="searchPatternEdit">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="searchNextPushButton">
         <property name="text">
          <string>Next</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="searchUpRadioButton">
         <property name="text">
          <string>up</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="searchDownRadioButton">
         <property name="text">
          <string>down</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="fltrFrame">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="fltrEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="addFltrButton">
             <property name="text">
              <string>Add filter</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListView" name="filtrListView">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="rmFltrButton">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Time range</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>From:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QTimeEdit" name="rangeFrom"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>To:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QTimeEdit" name="rangeTo"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
      <zorder>rmFltrButton</zorder>
      <zorder>groupBox</zorder>
      <zorder>filtrListView</zorder>
      <zorder>fltrEdit</zorder>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionfont_selection"/>
    <addaction name="actionclearSettings"/>
    <addaction name="separator"/>
    <addaction name="actionHighlights"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionreolad_current"/>
    <addaction name="actionclose_current_tab"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionexit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionshow_toolbar"/>
    <addaction name="actionautoscroll_enabled"/>
    <addaction name="actionEnable_filtration"/>
    <addaction name="actionselect_text_encoding"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionFind"/>
    <addaction name="actionsearch_end"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menu"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionreolad_current"/>
   <addaction name="actionOpen"/>
   <addaction name="actionclose_current_tab"/>
   <addaction name="actionautoscroll_enabled"/>
   <addaction name="actionFind"/>
   <addaction name="actionfont_selection"/>
   <addaction name="actionEnable_filtration"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/open.svg</normaloff>:/icons/open.svg</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>open file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionreolad_current">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/update.svg</normaloff>:/icons/update.svg</iconset>
   </property>
   <property name="text">
    <string>Reload all</string>
   </property>
   <property name="toolTip">
    <string>reolad current</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionclose_current_tab">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/close.svg</normaloff>:/icons/close.svg</iconset>
   </property>
   <property name="text">
    <string>Close current tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionautoscroll_enabled">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/autoscroll.svg</normaloff>:/icons/autoscroll.svg</iconset>
   </property>
   <property name="text">
    <string>Autoscroll enabled</string>
   </property>
   <property name="toolTip">
    <string>autoscroll</string>
   </property>
  </action>
  <action name="actionfont_selection">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/fonts.svg</normaloff>:/icons/fonts.svg</iconset>
   </property>
   <property name="text">
    <string>Select font</string>
   </property>
  </action>
  <action name="actionclearSettings">
   <property name="text">
    <string>Clear settings</string>
   </property>
  </action>
  <action name="actionshow_toolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show toolbar</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionselect_text_encoding">
   <property name="text">
    <string>Select text encoding</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/find.svg</normaloff>:/icons/find.svg</iconset>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionsearch_end">
   <property name="text">
    <string>search end</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionHighlights">
   <property name="text">
    <string>Highlights</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Recent files</string>
   </property>
  </action>
  <action name="actionEnable_filtration">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/filter.svg</normaloff>:/icons/filter.svg</iconset>
   </property>
   <property name="text">
    <string>Enable filtration</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionexit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchPatternEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>searchNextPushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>646</x>
     <y>443</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fltrEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>addFltrButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>741</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
